### DISCUSSION TITLE
是否可以终止对.NET 4.0和.NET 4.5的支持？

### DISCUSSION THREAD

**Discussion by wzd24** [2881099/FreeRedis#104] [General]
现在用这两个版本的项目应该已经很少了吧？我想贡献代码，但目前已经没有4.0和4.5的SDK下载了。

**Comment by 2881099** [comment]
.net4 还有不少用户，FreeSql 那边可以证明需求量，你可以先屏蔽 TargetFrameworks，编译测试通过后，我这边再合并代码做 .net4 适配
